purchase_operating_unit/tests/test_po_security.py

```diff
@@ -0,0 +1,41 @@
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
```

Comment:
isnot it redundant either with copyright or contributor's list?

Reply:
I don't know, but I prefer to not delete it just in case

Reply:
@jbeficent ?